samples/javascript_nodejs/06.using-cards/package.json

```diff
@@ -18,7 +18,6 @@
     "dependencies": {
         "botbuilder": "^4.3.0",
```

Comment:
aahhh 4.3.2

Reply:
... and then we will  be releasing with 4.4.0

Reply:
@johnataylor - What shall I change this to? It looks like 4.3.4 is available

Reply:
OK I guess 4.3.4 on node and 4.3.2 on C#.
Basically be on teh latest

Reply:
Done